lib/environment.js

```diff
@@ -32,6 +35,9 @@ exports.Environment = Environment = function(options) {
    delete options.tags;
    delete options.debug;
    this.config = options;
+   
+   this.resourcesDir = this.loader && typeof(this.loader.findResourcesDir) === "function" ? this.loader.findResourcesDir() : "./resources";
```

Comment:
Yeah, I agree, it's not quite right yet. Maybe you help me find a better way? Here's what I'm trying to accomplish. The idea is that each set of templates should come with its own resources. This should be true of mounted applications as well as templates loaded via a different loader. 

Advantages: 
* you can easily add or subtract sets of templates or mounted apps. Each set comes with everything it needs for translation. 
* sets and mounted app can easily be shared
* any loc tool that extracts strings for a set of templates only needs to scan that set of templates, not anything else.